### PR TITLE
fix(docs): incorrect type in withThread

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: etc/hub.api.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ```ts
 
+import type { AbortSignal as AbortSignal_2 } from 'abort-controller';
```
Comment (carsonfarmer) [inline etc/hub.api.md:7]: why is it doing this?

**Review by carsonfarmer — APPROVED** [review]
Looks good. The auto-generated code seems fishy though?

File: etc/hub.api.md (hunk 1 above)
Comment (andrewxhill) [reply etc/hub.api.md:7]: i'm guessing you didn't rebuild this file with your older abort work

Comment (carsonfarmer) [reply etc/hub.api.md:7]: I did not, but it seems strange that it is importing as AbortSignal_2? It does the same thing with ThreadId

Comment (andrewxhill) [reply etc/hub.api.md:7]: maybe it just doesn't understand the `import type` system well. but related, should AbortSignal be exported to end users actually?

Comment (carsonfarmer) [reply etc/hub.api.md:7]: We could export it yeh, its really just a import away, but easy enough to include it. Thing is, browsers already have that API, which is why we didn't include it by default. But yeh, can't hurt.

**Pull request merged**